test/compress/issue-1001.js

```diff
@@ -0,0 +1,8 @@
+parenthesis_strings_in_parenthesis: {
+    input: {
+        ('(');
+        a(')');
+
+    }
+    expect_exact: '"(";a(")");'
```

Comment:
I see you mean `("string");` shouldn't be a directive. I suppose, but it seems harmless enough. Unlikely for such code to be in the wild.